<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FTP</class>
 <widget class="QMainWindow" name="FTP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FTP</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>741</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="SendBtn">
         <property name="text">
          <string>开始发送</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="RecvBtn">
         <property name="text">
          <string>开始接收</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>端口号：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="PortlineEditSend"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>文件名：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="fileNamelineEditSend"/>
           </item>
           <item>
            <widget class="QPushButton" name="ChooseFileBtn">
             <property name="text">
              <string>选择文件</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>传输进度：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="SendprogressBar">
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>IP地址：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="IPlineEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>端口号：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="PortlineEditRecv"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>文件名：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="fileNamelineEditRecv"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>接收进度：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="RecvprogressBar">
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
